lib/workers/repository/update/branch/get-updated.ts

```diff
@@ -389,3 +395,23 @@ export async function getUpdatedPackageFiles(
     artifactErrors,
   };
 }
+
+// workaround, see #27319
+function patchConfigForArtifactsUpdate(
+  config: BranchConfig,
+  manager: string,
+  packageFileName: string,
+): BranchConfig {
+  const updatedConfig = { ...config };
+  if (is.nonEmptyArray(updatedConfig.packageFiles?.[manager])) {
+    const managerPackageFiles: PackageFile[] =
+      updatedConfig.packageFiles?.[manager];
+    const _packageFile: PackageFile | undefined = managerPackageFiles.find(
```

Comment:
```suggestion
    const packageFile = managerPackageFiles.find(
```
type should be inferred by typescript